KlingonAssistant/data/mem-05-H.xml

```diff
@@ -925,7 +925,7 @@ A physical path would be a {taw:n}.</column>
       <column name="entry_name">Heghba'</column>
       <column name="part_of_speech">n:alt,fic</column>
       <column name="definition">{Heghbat:n}</column>
-      <column name="definition_de"></column>
+      <column name="definition_de">{Heghbat:n}</column>
```

Comment:
I will pay attention to that in the future and will remove them all (since I did update all of those :- )